config/locales/react/react.sv.yml

```
@@ -4,3 +4,11 @@ sv:
       text_part_before: Ditt årliga klimatavtryck är
       text_part_middle: CO2e. För
       text_part_end: hjälper vi dig att klimatkompensera ditt avtryck. Summan fördelas till olika klimatprojekt som gör skillnad för klimatet.
+    memberships: 
+      free_title: "Gratis konto"
+      free_desc: "Spara och följ ditt klimatavtryck över tid"
+      single_title: "Kompensera månadsvis"
+      single_desc: "Stöd klimatprojekt motsvarande ditt klimatavtryck varje månad"
+      multi_title: "Kompensera för flera personer"
+      multi_desc: "Stöd klimatprojekt motsvarande %{num} gånger ditt klimatavtryck varje månad"
+      
```

@e-ry Hmm undra om det är pga detta? Det behöver inte vara "" runt :) 
```suggestion
    memberships: 
      free_title: Gratis konto
      free_desc: Spara och följ ditt klimatavtryck över tid
      single_title: Kompensera månadsvis
      single_desc: Stöd klimatprojekt motsvarande ditt klimatavtryck varje månad
      multi_title: Kompensera för flera personer
      multi_desc: Stöd klimatprojekt motsvarande %{num} gånger ditt klimatavtryck varje månad
      
```